Quest System/Unity Demo/Assets/QSystem/Assets/Scripts/QEnums.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace QSystem
+{
+    public enum PACKAGE_TYPE { Normal, Enveloppe, Medium, Can, Large }
+
+    public enum REGION_TYPE { Unkown, Rich, Poor, Industrial }
+
+    public enum CRITERIA_TYPE { Delivery, PickUp }
+
+    public enum REWARD_TYPE { Resource, Item }
+
+    public enum RESOURCE_TYPE { Money, Gold, Silver }
+
+    public enum QUEST_STATUS { Default, Accept, Complete, Fail }
+
+    public enum CHALLENGE_TYPE { Easy, Medium, Hard }
+
+    public enum LEGAL_STATUS { Unkown, Legal, Illegal }
+}
```

Comment:
Enum values doen we net als constants met: HOOFLETTERS, OF_ZO_MET_MEERDERE_WOORDEN